extensions/wikia/RobotsTxt/tests/WikiaRobotsTest.php

```diff
@@ -59,6 +60,20 @@ private function getRobotsTxtMock() {
 		return new RobotsTxtMock();
 	}
 
+	/**
+	 * Get Wikia RobotsTxt
+	 * @return RobotsTxt
+	 */
+	private function getWikiaRobotsTxt() {
+		$robotsTxtMock = $this->getRobotsTxtMock();
+		$pathBuilderMock = $this->getPathBuilderMock();
+
+		$wikiaRobots = new WikiaRobots( $pathBuilderMock );
+		$robotsTxt = $wikiaRobots->configureRobotsBuilder( $robotsTxtMock );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$robotsTxt". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)